src/components/App.tsx

```diff
@@ -1995,10 +1994,13 @@ class App extends React.Component<AppProps, AppState> {
             isTextElement(selectedElement) ||
             isValidTextContainer(selectedElement)
           ) {
+            const container = selectedElements[0] as ExcalidrawTextContainer;
```

Comment:
this isn't correct. `selectedElement` can be a text element (= not a container) as per above condition.